iis-core/src/test/java/eu/dnetlib/iis/core/AbstractOozieWorkflowTestCase.java

```diff
@@ -61,13 +83,34 @@
 	@BeforeClass
 	public static void classSetUp() {
 		propertiesReader = new IntegrationTestPropertiesReader();
+		
+		
 	}
 	
 	@Before
-	public void setUp() {
+	public void setUp() throws IOException {
 		
 		log.debug("Setting up OozieClient at {}", getOozieServiceLoc());
 		oozieClient = new OozieClient(getOozieServiceLoc());
+		
+		
+		String s = "<?xml-stylesheet type=\"text/xsl\" href=\"configuration.xsl\"?>"
+				+ "<configuration>"
+				+ "<property><name>fs.defaultFS</name><value>" + getNameNode() + "</value></property>"
+				+ "</configuration>";
+		
+		
+		Configuration hdfsConf = new Configuration(false);
+		hdfsConf.addResource(new ReaderInputStream(new StringReader(s)));
+		
```

Comment:
Hasn't it been possible to use something like hdfsConf.set("fs.defaultFS", getNameNode())?